model/app.py

```diff
@@ -218,7 +218,7 @@ def key_from_model_info(wild_type_id, message):
     :param message: dict
     :return: str
     """
-    d = {k: message.get(k, []) for k in {GENOTYPE_CHANGES, MEDIUM, MEASUREMENTS}}
```

Comment:
added/removed reactions can be a) defined by the genotype, in this case info is already there; b) happen in "runtime" through the websocket, in this case they are not part of this initial message
so I think it's not needed to use it in the key
(but it reminds me to maybe think about making a more clear distinction between "prior" and "runtime" changes, so runtime ones can only be made through the websocket and not through the post request)